src/plugins/numberline/components/numberline-tile.tsx

```diff
@@ -328,11 +381,37 @@ export const NumberlineTile: React.FC<ITileProps> = observer(function Numberline
         style={{"height": `${kNumberLineContainerHeight}`}}
       >
         <div className="numberline-tool-container" >
-          <i className="arrow left" style={{ left: xShiftNum + kArrowheadOffset, top: kArrowheadTop }}/>
-          <i className="arrow right" style={{ right: xShiftNum + kArrowheadOffset, top: kArrowheadTop }}/>
           <svg ref={svgRef} width={axisWidth}>
             <g ref={axisRef}></g>
           </svg>
+          <NumberlineArrowLeft
+            className="arrow"
+            style={{ left: arrowOffset, top: kArrowheadTop }}
+          />
+          <NumberlineArrowRight
+            className="arrow"
+            style={{ right: arrowOffset, top: kArrowheadTop }}
+          />
+          {
+            <>
+              <EditableNumberlineValue
+                value={content.min}
+                minOrMax={"min"}
+                offset={arrowOffset}
+                readOnly={readOnly}
+                isTileSelected={isTileSelected}
+                onValueChange={(newValue) => handleMinMaxChange("min", newValue)}
+              />
+              <EditableNumberlineValue
+                value= {content.max}
+                minOrMax={"max"}
+                offset={arrowOffset}
+                readOnly={readOnly}
+                isTileSelected={isTileSelected}
+                onValueChange={(newValue) => handleMinMaxChange("max", newValue)}
+              />
+            </>
+          }
```

Comment:
```suggestion
              <EditableNumberlineValue
                value={content.min}
                minOrMax={"min"}
                offset={arrowOffset}
                readOnly={readOnly}
                isTileSelected={isTileSelected}
                onValueChange={(newValue) => handleMinMaxChange("min", newValue)}
              />
              <EditableNumberlineValue
                value= {content.max}
                minOrMax={"max"}
                offset={arrowOffset}
                readOnly={readOnly}
                isTileSelected={isTileSelected}
                onValueChange={(newValue) => handleMinMaxChange("max", newValue)}
              />
```
I don't think you need the containing elements here.